kibana/server.sls

```diff
@@ -1,21 +1,15 @@
 {%- from "kibana/map.jinja" import server with context %}
+{%- from "linux/map.jinja" import system with context %}
 {%- if server.enabled %}
 
-{%- if server.addrepo is defined and grains['os_family'] == 'Debian' %}
```

Comment:
Formulas should not do what they are not supposed to do so it's good that you removed this. Anyway default metadata should not contain this definition. Move it into separate metadata/service file or remove completely.